### PR TITLE
[Orc8r] 5G WS#2-MVC1 User Guide

### PR DESCRIPTION
**Pull request by GANESH-WAVELABS**
Signed-off-by: GANESH IRRINKI <ganesh.irrinki@wavelabs.ai>

## Summary

OCN 5G WS#2 - Orchestration MVC1(Monitoring stack for 5G Core Containers)
Kibana Dashboard configured to view AGW Syslogs.
Script to deploy K8s Cluster, Monitoring Stack, AGW VM (Magma Dev)
User Guide/README document for deployment steps - K8s Cluster, Monitoring Stack, AGW VM (Magma Dev)

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: cn/deploy/scripts/MVC1_5G_Orc8r_deployment_script.py [hunk 1]
```diff
@@ -0,0 +1,379 @@
...
+import webbrowser, socket
+import threading, platform
+
+
+#Initialize the lock
+lock = threading.Lock()
+#Dictionary to maintain k8s services whether they are Running or not
+k8s_obj_dict = {}
+#Build context for Docker files for AMF, SMF, UPF
+BUILD_CONTEXT_PARENT_DIR = '/tmp/Orc8r_docker'
```
Comment (mpgermano) [inline cn/deploy/scripts/MVC1_5G_Orc8r_deployment_script.py]: nit: lets name this `/tmp/orc8r_docker`

File: cn/deploy/scripts/MVC1_5G_Orc8r_deployment_script.py [hunk 2]
```diff
@@ -0,0 +1,379 @@
+#!/usr/bin/env python3
```
Comment (mpgermano) [inline cn/deploy/scripts/MVC1_5G_Orc8r_deployment_script.py]: Ni: can we rename this file `mvc1_5g_orc8r_deployment_script.py`

**Review by mpgermano — CHANGES_REQUESTED** [review]
This looks good. Can you please address the static analysis issues. Then I can approve.

**Comment by GANESH-WAVELABS** [comment]
Hi Michael,

Thanks for the review for the PR(#4515), I have modified the code according to your two comments/suggestions.

1)  renamed the file  "MVC1_5G_Orc8r_deployment_script.py" to "mvc1_5g_orc8r_deployment_script.py"
2) renamed the file '/tmp/Orc8r_docker' to '/tmp/orc8r_docker'

Also fixed all the possible staic analysis issues. Could you please review it and help me to approve & merge this PR.

Thanks & Regards,
Ganesh . Irrinki,
Email-Id : ***@***.*** ,
Mobile : +91 8121666196, +91 8867839739,
Wavelabs Technologies PVT LTD
________________________________

**Review by mpgermano — APPROVED** [review]
Looks good. Thanks for this.

**Comment by electronjoe** [comment]
There are two new tests in the CI pipeline (well not new tests, but newly running in CI and newly blocking merges).  Can you rebase and push so that CI runs?  The tests are the following.

```
ci/circleci: mme_test
ci/circleci: session_manager_test
```

[Edit] These tests will not actually have to run because no c/cpp is touched, but Github requires them to be evaluated as un-necessary for the merge to occur, which requires the CircleCI test be defined in your branch - which requires the rebase.

**Pull request merged**